input/fsh/VRDR/newinstances/EX_BirthRecordIdentifier.fsh

```diff
@@ -0,0 +1,35 @@
+Instance: BirthRecordIdentifier-Example1
+InstanceOf: BirthRecordIdentifier
+Usage: #example
+Description: "BirthRecordIdentifier-Example1"
+* insert AddMetaProfile(BirthRecordIdentifier)
+* status = #final
+//* code = $v2-0203#BR "Birth registry number"
+* subject = Reference(Decedent-Example1)
+* valueString = "717171"
+* component[birthJurisdiction].valueString = "YC"
+* component[birthYear].valueDateTime = "1961"
```

Comment:
I might be a bit confused on this, maybe you could clarify for me. Including a birthRecordIdentifier is "required only when the decedent is an infant." This means that the birth record ID here is related to when the subject was born, correct? So, in theory, this example does not need to be included (but may as well be). I might have been confused about this because the value of the ID is the same as that for the fetal death/child birth record ID's and include the birth year of the mother, not her child. My understanding is the below examples meant to be links to the records of the birth of the decedent's child.

Reply:
Also, decedent-example1's birthdate year is 2004 and the birth record identifier birth year is 1961